template/build.cmd

```diff
@@ -0,0 +1,17 @@
+@ECHO off
+SETLOCAL
+
+SET BUILD_FOLDER=.build
+
+SET BOOTSTRAP_FILE_REMOTE=https://raw.githubusercontent.com/aspnet/KoreBuild/victorhu/init/bootstrap/build.cmd
+SET BUILD_FILE=%BUILD_FOLDER%\build.cmd
+
+SET REPO_FOLDER=%~dp0
+CD "%REPO_FOLDER%"
+
+if EXIST %BUILD_FILE% GOTO :EndBootstrapGet
+mkdir %BUILD_FOLDER%
+@powershell -NoProfile -ExecutionPolicy unrestricted -Command "$ProgressPreference = 'SilentlyContinue'; Invoke-WebRequest '%BOOTSTRAP_FILE_REMOTE%' -OutFile '%BUILD_FILE%'"
```

Comment:
Same suggestion as above. Why not just do this in powershell and use the 2-line cmd file to launch a ps1 instead? CMD is ugly IMO